resources/qml/ReplyPopup.qml

```diff
@@ -44,4 +46,17 @@ Rectangle {
         onClicked: room.reply = undefined
     }
 
+    Button {
+        id: closeEditButton
+
+        visible: room && room.edit
+        anchors.left: parent.left
+        anchors.rightMargin: 16
+        anchors.topMargin: 10
+        anchors.top: parent.top
+        //height: 16
+        text: qsTr("Abort edit")
```

Comment:
Is 'Abort edit' an official phrase from somewhere in the spec?  Otherwise, I'd probably use 'Cancel edit' instead.  'Abort' has a sort of negative connotation associated with it.